### PR TITLE
Feature/add registration screen

### PR DESCRIPTION
**Pull request by RickWoltheus**
This pr adds the following
- Aanmelding screen
- Aanmelding tab
- Several refactors for fieldsets

Note: data is still stubbed

### CODE REVIEW

File: client/src/components/fieldsets/participants/fieldsets/RegistratorInformationFieldset.tsx [hunk 1]
```diff
@@ -59,36 +68,40 @@ const RegistratorInformationFieldset: React.FunctionComponent<Props> = props =>
...
+                    <DateInput
+                        name="date"
+                        placeholder={i18n._(t`01/01/2020`)}
+                        defaultValue={prefillData?.date ?? undefined}
+                    />
                 </Field>
                 <Field label={i18n._(t`Aanmeldende instantie`)} horizontal={true}>
                     <Input
                         name="registering-party"
                         placeholder={i18n._(t`1234 AB`)}
```
Comment (bbawuah) [inline client/src/components/fieldsets/participants/fieldsets/RegistratorInformationFieldset.tsx:80]: I made a mistake in this Input. I'll fix this in another PR

Comment (RickWoltheus) [reply client/src/components/fieldsets/participants/fieldsets/RegistratorInformationFieldset.tsx:80]: What mistake is present here? isn't it fixed here?

**Review by bbawuah — APPROVED** [review]

File: client/src/components/fieldsets/participants/fieldsets/RegistratorInformationFieldset.tsx (hunk 1 above)
Comment (bbawuah) [reply client/src/components/fieldsets/participants/fieldsets/RegistratorInformationFieldset.tsx:80]: The input name 'registering-party' is not the same as in the fieldsetModel. It also has a wrong placeholder and validator

Comment (RickWoltheus) [reply client/src/components/fieldsets/participants/fieldsets/RegistratorInformationFieldset.tsx:80]: Ah oke, then i will merge this, so you can use my changes ;)

**Pull request merged**